src/sails.ts

```diff
@@ -126,7 +126,13 @@ export class Sails implements SailsInterceptorInterface, SailsInterceptorHandler
     }
 
     public on(eventName: string, cb) {
```

Comment:
@burntblark why is the return types different for `on` and `off`?

Reply:
I have not figured out what `off` does exactly so I never changed its implementation.

Reply:
Okay. I'm still waiting for your response on #29.